chemie/events/forms.py

```diff
@@ -144,6 +155,7 @@ class RegisterEventForm(BaseRegisterEventForm):
         M.Row("price_member", "price_not_member", "price_companion"),
         M.Row("companion", "sleepover", "night_snack", "check_in"),
         M.Row("allowed_grades"),
+        M.Row("allowed_groups")
```

Comment:
This row shows up even when there are no existing groups
![image](https://user-images.githubusercontent.com/35535819/108173534-862c8200-70fe-11eb-841c-e800a5d0bbad.png).
Is there a way to prevent this?

Reply:
Good catch! I will look into it. For later, the ideal scenario is of couse to not have checkboxes, but a list or something to display the groups, or a button with "add group".

Reply:
Hmm the fix for this is a little more complicated to resolve than I thought. For now, there will exist a Kohortsjefer group, so I propse we look further into this issue when we are making the functionality more general later? Then we can deploy the functionality today :-)